### PR TITLE
moved hard-coded admin

### PR DESCRIPTION
**Pull request by jiayiw6**
Changed when to add admins , so activity feed can function better with admins. Resolves both @issue55 and @issue24

### CODE REVIEW

File: src/main/java/codeu/model/store/basic/UserStore.java [hunk 1]
```diff
@@ -57,7 +60,12 @@ public static UserStore getInstance() {
    * @param persistentStorageAgent a mock used for testing
    */
   public static UserStore getTestInstance(PersistentStorageAgent persistentStorageAgent) {
-    return instance = new UserStore(persistentStorageAgent);
+    instance = new UserStore(persistentStorageAgent);
+    instance.setActivityStore(ActivityStore.getTestInstance(persistentStorageAgent));
+
+    // hard-coded initial Admin:
+    instance.addUser("Admin01", "AdminPass01", /* admin= */ true);
```
Comment (anthony23334) [inline src/main/java/codeu/model/store/basic/UserStore.java]: Why do we need the admin when running tests? I was thinking that it was only necessary for the non-test part.

File: src/main/java/codeu/model/store/basic/UserStore.java [hunk 2]
```diff
@@ -71,12 +79,7 @@ public static UserStore getTestInstance(PersistentStorageAgent persistentStorage
   /** This class is a singleton, so its constructor is private. Call getInstance() instead. */
   private UserStore(PersistentStorageAgent persistentStorageAgent) {
     this.persistentStorageAgent = persistentStorageAgent;
-    // Todo: This should be inside getTestInstance() once we fixed line 79
-    this.setActivityStore(ActivityStore.getTestInstance(persistentStorageAgent));
     users = new ArrayList<>();
```
Comment (anthony23334) [inline src/main/java/codeu/model/store/basic/UserStore.java]: Perfect!

File: src/main/java/codeu/model/store/basic/UserStore.java (hunk 1 above)
Comment (jiayiw6) [reply src/main/java/codeu/model/store/basic/UserStore.java]: There are actually tests for the initial admin to check if they are added when creating a new UserStore

File: src/main/java/codeu/model/store/basic/UserStore.java (hunk 2 above)
Comment (jiayiw6) [reply src/main/java/codeu/model/store/basic/UserStore.java]: More activity feed stuff! YAY!

File: src/main/java/codeu/model/store/basic/UserStore.java (hunk 1 above)
Comment (anthony23334) [reply src/main/java/codeu/model/store/basic/UserStore.java]: screen shot 2018-06-29 at 1 40 37 pm
I don't think it is necessary to add that into getTestInstance(). I just removed it and this is the only error I got. It is a normal error.

Comment (jiayiw6) [reply src/main/java/codeu/model/store/basic/UserStore.java]: Yeah, so it's testing if the hard-coded admin is added. It's expecting 3(1 initial admin + 2 later added) and it's getting 2(no initial admin + 2 later added). I feel like the point of testing is to see if it's behaving as expected (we are expecting 1 initial admin), so I kept the test instance matching with the real instance. 
BTW, if you change that 3 to a 2 in the testing, you will get an assert error later (that was the error I was getting last night)
What do you think?

Comment (anthony23334) [reply src/main/java/codeu/model/store/basic/UserStore.java]: The error is coming from the boolean check you added inside the for loop to check for admin. But I think you are right about testing that it is actually creating another admin in the back. Yeah, we should keep it then as long as everything else is working

**Pull request merged**